src/VoltServiceProvider.php

```diff
@@ -61,8 +62,10 @@ protected function bindLivewireManager(): void
     {
         $this->app->singleton(LivewireManager::class);
         $this->app->alias(LivewireManager::class, 'livewire');
+        $this->app->alias(LivewireManager::class, BaseLivewireManager::class);
 
         Facade::clearResolvedInstance('livewire');
+        Facade::clearResolvedInstance(BaseLivewireManager::class);
```

Comment:
Fixed integration with Livewire 3.7 and 4.0: https://github.com/livewire/livewire/pull/9220